modules/createServerRenderContext.js

```diff
@@ -1,34 +1,36 @@
-const k = () => {}
-
 const createServerRenderContext = () => {
   let flushed = false
+  let flushedIndex = 0
   let redirect = null
   let matchContexts = [
-    /* { hasMissComponent: bool, matchesByIdentity: [] } */
+    /* { hasMissComponent: bool, hasMatches: bool } */
   ]
 
-  const setRedirect = flushed ? k : (location) => {
```

Comment:
I'm not entirely sure what the idea with these ternary statements are. They executewhen `createServerRenderContext` is called, at which point `flushed === false`, so they are only ever the second option.